Tests/RemoteDumpTest.php

```diff
@@ -244,13 +244,8 @@ public function addAdditionalOptionsToRequest()
         $request = $this->runProtectedMethod('getConfiguredHttpRequest', []);
         $options = $request->getOptions();
 
-        if (empty($options['stream']) || empty($options['headers']['Accept'])) {
-            $result = false;
-        } else {
-            $result = true;
-        }
-
-        $this->assertTrue($result);
+        $this->assertTrue(isset($options['stream']));
+        $this->assertTrue(isset($options['headers']['Accept']));
```

Comment:
if possible in a simple manner, check for the content if that proves anything

Reply:
Done.